vlingo-router-contentbased/src/main/java/io/vlingo/router/contentbased/order/OrderItem.java

```diff
@@ -0,0 +1,57 @@
+package io.vlingo.router.contentbased.order;
+
+/**
+ * @author Chandrabhan Kumhar
+ * Stores details for a specific order
+ */
+public class OrderItem {
```

Comment:
On this platform and in general I completely avoid the use of the Java Bean spec. Besides, I believe that the `OrderItem` is always set-once read-many, so all you need is a constructor that takes all attributes and sets them on `final` instance variables.